src/services/expenses.service.js

```diff
@@ -0,0 +1,69 @@
+const { Expense } = require('../models/Expense.model.js');
+
+const normalize = ({ id, userId, spentAt, title, amount, category, note }) => {
+  return {
+    id,
+    userId,
+    spentAt,
+    title,
+    amount,
+    category,
+    note,
+  };
+};
+
+async function getAll() {
+  const result = await Expense.findAll();
+
+  return result;
+}
+
+async function getOne(id) {
+  return Expense.findByPk(id);
+}
+
+async function createOne({ userId, spentAt, title, amount, category, note }) {
+  return Expense.create({
+    userId,
+    spentAt,
+    title,
+    amount,
+    category,
+    note,
+  });
+}
```

Comment:
```suggestion
async function createOne(expense) {
  return Expense.create(expense);
}
```
Looks like you don't need destructing